tasks/main.yml

```
@@ -19,22 +19,29 @@
   ansible.builtin.script: "{{ __snapshot_cmd }}"
   args:
     executable: "{{ __snapshot_python }}"
-  register: snapshot_cmd
+  register: snapshot_cmd_raw
   no_log: true
   ignore_errors: true  # noqa: ignore-errors - handled below
+  changed_when: false  # change handled below too
 
 - name: Print out response
   debug:
-    var: snapshot_cmd.stdout
+    var: snapshot_cmd_raw
```

```suggestion
    var: snapshot_cmd_raw.stdout
```
Should it print only stdout as it was before?

---

No - would prefer to see all of the fields rather than just `stdout`